subtrack.MAUI/Pages/CreateSubscription.razor

```diff
@@ -1,13 +1,36 @@
 ﻿@page "/CreateSubscription"
 @inject NavigationManager NavigationManager
 
-<h3>CreateSubscription</h3>
-<br/>
-<p>returnUrl: @ReturnUrl</p>
+<div style="display: flex; justify-content: space-between; padding-bottom: 20px;">
+    <button type="button" class="btn btn-outline-dark">Back</button>
+    <button type="button" class="btn btn-outline-dark">Save</button>
+</div>
+
+<EditForm Model="@subscription">
+    <div>
+        <InputText class="form-control" placeholder="Service Name" @bind-Value="@subscription.Name"></InputText>
+    </div>
+    <div class="input-group mb-3" style="padding-top: 15px;">
+        <span class="input-group-text">$</span>
```

Comment:
Would be nice if this was not hardcoded but rather used the user's currency. Some easy ways mentioned here https://stackoverflow.com/questions/2763128/get-the-currency-from-current-culture